core/modules/parsers/wikiparser/rules/prettylink.js

```diff
@@ -49,13 +49,12 @@ exports.parse = function() {
 		}];
 	} else {
 		return [{
-			type: "link",
-			attributes: {
-				to: {type: "string", value: link}
-			},
-			children: [{
-				type: "text", text: text
-			}]
+			type: "macrocall",
```

Comment:
The problem now is, that backlinks don't work anymore :)  ... That's the exact same behavior, that unilink-plugin uses. 

I did use an [ugly hack](https://github.com/wikilabs/plugins/blob/master/wikilabs/uni-link/tiddlers/modules/uni-link.js#L84) so I didn't need to modify the core backlink handling. 

alias-handling has it's own [backlink functions](https://github.com/wikilabs/plugins/blob/master/wikilabs/uni-link/tiddlers/modules/wiki-methodes.js)

macros can be found [here](https://github.com/wikilabs/plugins/tree/master/wikilabs/uni-link/tiddlers): uni-link-macro.tid, aka-macros.tid ...

Reply:
I think we can address that by updating the link detection code to also look for the corresponding macro.

Reply:
yea, I know. ... but for the plugin, I didn't want to mess with the core code.